typescript/test/test_application_js/package.json

```diff
@@ -6,7 +6,7 @@
     "browser-test": "karma start"
   },
   "dependencies": {
-    "showcase": "file:///../../../.baseline-test-out",
+    "showcase": "file:./showcase-0.1.0.tgz",
     "@grpc/grpc-js": "^0.5.2",
     "grpc": "^1.23.3",
```

Comment:
`grpc` is not used in this test application.

Reply:
removed it